app/components/editUserProfile.jsx

```diff
@@ -0,0 +1,379 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import OrderSummary from './OrderSummary';
+import { FormHelperText } from '@material-ui/core';
+import { Button, TextField, Grid, Typography } from '@material-ui/core';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { modifyUser } from '../redux/activeUser';
+class EditUserProfile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      shippingAddress1: '',
+      shippingAddress2: '',
+      shippingCity: '',
+      shippingState: '',
+      shippingZip: '',
+      billingAddress1: '',
+      billingAddress2: '',
+      billingCity: '',
+      billingState: '',
+      billingZip: '',
+      password: '',
+      error: false,
+      errorText: ''
+    };
+  }
+
+  componentDidMount() {
+    const { activeUser } = this.props;
+    console.log('activeUser', activeUser);
+    this.setState({
+      firstName: activeUser.firstName,
```

Comment:
could probably just do this.setState(activUser)

Reply:
if this is coming from redux state then just use the redux state and dispatches
action creator: changeValue({key: value})
reducer: return {...state, ...changedValue}